internal/appsec/dyngo/instrumentation/httpinstr/http.go

```diff
@@ -89,18 +86,16 @@ func WrapHandler(handler http.Handler, span ddtrace.Span) http.Handler {
 		}
 		host := r.Host
 		headers["host"] = []string{host}
+		var secEvent json.RawMessage
 		op := StartOperation(
 			HandlerOperationArgs{
-				Span:       span,
-				IsTLS:      r.TLS != nil,
-				Method:     r.Method,
-				Host:       r.Host,
-				Path:       r.URL.Path,
+				OnSecurityEvent: func(event json.RawMessage) {
+					secEvent = event
```

Comment:
Very simple handler because the WAF is called once per request handler for now: it simply saves it until the end of the request so that the appsec span tags end up being set when it's not nil in the return path.